src/bin/wasi-virt.rs

```diff
@@ -58,6 +58,15 @@ struct Args {
     #[arg(short, long, use_value_delimiter(true), value_name("ENV=VAR"), value_parser = parse_key_val::<String, String>, help_heading = "Env")]
     env: Option<Vec<(String, String)>>,
 
+    // CONFIG
+    /// Allow unrestricted access to host configuration properties, or to a comma-separated list of property names.
```

Comment:
Perhaps let's be explicit that this is the `runtime-config` system by including this term in the help?